contextMenu.js

```diff
@@ -135,6 +136,26 @@
                             }
                         }
 
+						//update ico
+                        if (ico) {
+                            var imgIcon = elm.find('.iw-mIcon');
+                            if (imgIcon.length) {
+								if(imgIcon[0].tagName.toLowerCase() === "span"){
+									var classList = ["iw-mIcon"];
```

Comment:
I didn't thing of adding multiple classes as string with spaces. That simplifies stuff a lot. 👍